rs/nervous_system/integration_tests/src/pocket_ic_helpers.rs

```diff
@@ -214,29 +212,120 @@ pub async fn install_canister_on_subnet(
     CanisterId::unchecked_from_principal(canister_id.into())
 }
 
+/// A builder for adding SNS WASW modules to SNS-W canister.
+#[derive(Default)]
+pub struct SnsWasmCanistersInstaller {
+    pub mainnet_sns_canister_versions: Option<bool>,
+    pub neuron_id: Option<NeuronId>,
+    pub principal_id: Option<PrincipalId>,
+}
+
+impl SnsWasmCanistersInstaller {
+    /// Requests the mainnet Wasm versions for all NNS canisters being installed.
+    pub fn with_mainnet_sns_canister_versions(&mut self) -> &mut Self {
+        self.mainnet_sns_canister_versions = Some(true);
+        self
+    }
+
+    /// Requests tip-of-this-branch Wasm versions for all NNS canisters being installed.
+    pub fn with_current_sns_canister_versions(&mut self) -> &mut Self {
+        self.mainnet_sns_canister_versions = Some(false);
+        self
+    }
+
+    /// Specifies the ID of the neuron which will be used to create NNS proposals and
+    /// principal ID that is able to submit proposals on behalf of the provided neuron.
+    /// If the neuron is not specified with this method, 'TEST_NEURON_1_ID' will be used
+    pub fn with_nns_neuron(&mut self, neuron_id: NeuronId, principal_id: PrincipalId) -> &mut Self {
+        self.neuron_id = Some(neuron_id);
+        self.principal_id = Some(principal_id);
+        self
+    }
+
+    pub async fn add_wasm_via_nns_proposal(
```

Comment:
Please add a doc string to indicate that this function defaults to using `TEST_NEURON_1_ID`